plugins/prometheus_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+file(GLOB HEADERS "include/eosio/prometheus_plugin/*.hpp")
+add_library( prometheus_plugin
+        prometheus_plugin.cpp
+        ${HEADERS} )
+
+target_link_libraries( prometheus_plugin appbase fc http_plugin chain_plugin eosio_testing_contracts )
+target_include_directories( prometheus_plugin PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include
```

Comment:
The includes and libraries need to be fixed up here. It looks like there were copied from somewhere else

Reply:
> Is it expensive to build or something? Optional items like this start raising some questions like:
> 
> * Should we be building it in CI?
>   
>   * Should we be building it both _on and off_ in CI?
> * Should it be enabled in binary builds we ship

I don't forsee it being expensive to build.  Do you want me to always include it?